libiocage/lib/Release.py

```diff
@@ -332,7 +376,7 @@ def fetch_updates(self):
                             "Components"
                         ))
                         f.truncate()
-                        f.close()
```

Comment:
Why are you removing this? I don't see it being used anywhere, this will result in leaking fd's

Reply:
`with` is already taking care of that. See http://effbot.org/zone/python-with-statement.htm

Reply:
I know how `with` works ;) I commented on that in a different ticket. I just forgot to click that little expand thing.